---
rapid_robot/Location[] locations
